k8s/aat/common/ccpay/ccpay-bulkscanning-app.yaml

```diff
@@ -33,8 +33,10 @@ spec:
         "ccpay":
           resourceGroup: ccpay
           secrets:
-          - bulkscanning-api-POSTGRES-PASS
-          - AppInsightsInstrumentationKey
+            - bulkscanning-api-POSTGRES-PASS
```

Comment:
These changes shouldn't go with this PR as it is a different application which is working in AAT